### ISSUE TITLE
bug(mat-autocomplete): listbox missing label

### ISSUE THREAD

**Issue by calebegg** [angular/components#20888]
The `<div role="listbox">` produced by the autocomplete should have an aria-label or aria-labelledby

example: https://www.w3.org/TR/wai-aria-practices/examples/combobox/aria1.1pattern/listbox-combo.html
